theme/layouts/_header.ejs

```diff
@@ -0,0 +1,27 @@
+<header>
+  <div class="tablet-hide">
+    <div class="col2">
+      <a href="\">
+        <img src="/images/icon.png" />
+        <p>
+        <%= site.title %>
+        </p>
+      </a>
+      <% include _language %>
+      <div class="cell1-2">
+        <% include _navigation %>
+      </div>
+    </div>
+  </div>
+  <div class="desktop-hide">
+    <div class="col2">
+      <a href="/" class="cell1 start">
+        <img src="/images/icon.png" alt="Privacy Manager logo">
```

Comment:
Detail: The higher quality logo can be found here -> https://github.com/Privacy-Managers/Privacy-Manager/blob/master/src/img/logo.png